pkg/tsdb/cloudwatch/log_sync_query.go

```diff
@@ -82,7 +80,7 @@ var executeSyncLogQuery = func(ctx context.Context, e *cloudWatchExecutor, req *
 }
 
 func (e *cloudWatchExecutor) syncQuery(ctx context.Context, logsClient cloudwatchlogsiface.CloudWatchLogsAPI,
-	queryContext backend.DataQuery, logsQuery models.LogsQuery) (*cloudwatchlogs.GetQueryResultsOutput, error) {
+	queryContext backend.DataQuery, logsQuery models.LogsQuery, logsTimeout string) (*cloudwatchlogs.GetQueryResultsOutput, error) {
```

Comment:
(see other comment, I believe you can set logsTimeout type can be time.Duration)